hack/table-gen/table-gen.go

```diff
@@ -0,0 +1,264 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	REPattern      = `(?s)<!--\s*TABLE-START\s* -->.*<!--\s*TABLE-END\s*-->` //Regular expression patter for reading everything between TABLE-START and TABLE-END tags
+	SkipIdentifier = `SKIP-ELEMENT`
+	RESkipPattern  = `<!--\s*` + SkipIdentifier + `\s*([^\s]+)\s*-->` // Regular expression pattern for skipping one element without children that was specified in MD
+	// TODO: change to skip with children
+	SkipWithAncestorsIdentifier = `SKIP-WITH-ANCESTORS`
+	RESkipWithAncestorsPattern  = `<!--\s*` + SkipWithAncestorsIdentifier + `\s*([^\s-]+)\s*-->` // Regular expression pattern for skipping an element with its children
+)
```

Comment:
Why in `RESkipPattern ` the capturing group is `([^\s]+)`, while in `RESkipWithAncestorsPattern ` the capturing group is `([^\s-]+)`? shouldn't the capturing group be the same in both regular expressions?

Reply:
I agree, I see no point of ignoring `-` here as well, will change it